ivy/functional/frontends/paddle/nn/functional/pooling.py

```diff
@@ -1 +1,65 @@
 # local
+import ivy
+from ivy.func_wrapper import with_supported_dtypes
+from ivy.functional.frontends.paddle.func_wrapper import to_ivy_arrays_and_back
+
+
+def _broadcast_pooling_helper(x, pool_dims: str = "2d", name: str = "padding"):
+    dims = {"1d": 1, "2d": 2, "3d": 3}
+    if isinstance(x, int):
+        return tuple([x for _ in range(dims[pool_dims])])
+    if len(x) == 1:
+        return tuple([x[0] for _ in range(dims[pool_dims])])
+    elif len(x) == dims[pool_dims]:
+        return tuple(x)
+    elif len(x) != dims[pool_dims]:
+        raise ValueError(
+            f"`{name}` must either be a single int, "
+            f"or a tuple of {dims[pool_dims]} ints. "
+        )
```

Comment:
Isn't this the same as the helper in `ivy/functional/frontends/torch/nn/functional/pooling_functions.py`? You can just import it from there.

Reply:
It is. Alright, makes sense 👍